master/jenkins_files/files/home/jenkins-slave/cleanup_docker_images.py

```diff
@@ -113,14 +113,14 @@ def main():
                         help='The minimum age of items to clean up in days.')
     parser.add_argument('--min-hours', type=int, default=10,
                         help='The minimum age of items to clean up in hours, added to days.')
-    parser.add_argument('--docker-api-version', type=str, default='1.16',
+    parser.add_argument('--docker-api-version', type=str, default='1.30',
```

Comment:
Note: this is what is on my workstation, may need to be updated according to what version the buildfarm machines are provisioned with